src/main/java/com/redhat/labs/lodestar/rest/client/LodeStarGDriveApiClient.java

```diff
@@ -0,0 +1,21 @@
+package com.redhat.labs.lodestar.rest.client;
+
+import javax.enterprise.context.ApplicationScoped;
+import javax.ws.rs.GET;
+import javax.ws.rs.Path;
+import javax.ws.rs.Produces;
+import javax.ws.rs.core.Response;
+
+import org.eclipse.microprofile.rest.client.annotation.RegisterClientHeaders;
+import org.eclipse.microprofile.rest.client.inject.RegisterRestClient;
+
+//@ApplicationScoped
```

Comment:
Not sure what happens here if this client is not hooked up and is called from the ArtifactResource.  Could be removed if not using or most likely nobody will actually try it.

Reply:
whoops. This file shouldn't be here. I'll fiix.